### PR TITLE
feat(prettier): remove eslint-plugin-prettier

### PR DESCRIPTION
**Pull request by jasonschroeder-sfdc**
It's not a recommended plugin, per the Prettier folks. And it does seem
to be slow.

Closes: #17
Closes: #19

### CODE REVIEW

**Review by RodEsp — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -282,7 +282,7 @@ escape-string-regexp@^4.0.0:
 
 eslint-config-prettier@^8.5.0:
   version "8.5.0"
-  resolved "https://registry.yarnpkg.com/eslint-config-prettier/-/eslint-config-prettier-8.5.0.tgz#5a81680ec934beca02c7b1a61cf8ca34b66feab1"
+  resolved "https://nexus-proxy-prd.soma.salesforce.com/nexus/content/groups/npm-all/eslint-config-prettier/-/eslint-config-prettier-8.5.0.tgz#5a81680ec934beca02c7b1a61cf8ca34b66feab1"
```
Comment (smithgp) [inline yarn.lock:285]: Sorry for the random drive-by review (saw this on slack), but I don't think you want this yarn.lock change in a public repo.

**Review by cristiand391 — CHANGES_REQUESTED** [review]
@maggiben noticed this first 👁️ , this should be using the yarn registry instead of nexus.

**Review by RodEsp — APPROVED** [review]
